controller.tf

```diff
@@ -58,22 +60,22 @@ data "external" "kubeadm_join" {
   program = ["./scripts/kubeadm-token.sh"]
```

Comment:
I had to change this to `${path.module}/scripts/kubeadm-token.sh`  to get it working

Reply:
Thanks @tabdulazim ! Added your suggestion.